src/include/sof/audio/component_ext.h

```diff
@@ -180,20 +180,12 @@ static inline int comp_copy(struct comp_dev *dev)
 
 	/* copy only if we are the owner of the component */
 	if (cpu_is_me(dev->ipc_config.core)) {
-/* BugLink: https://github.com/zephyrproject-rtos/zephyr/issues/43786
- * TODO: Remove this once the bug gets fixed.
- */
-#ifndef __ZEPHYR__
+#if CONFIG_PERFORMANCE_COUNTERS
```

Comment:
Is the above bug fixed by this commit ?

Reply:
not yet, let me fix that comp_copy first, then split patch for merge, according to songchao, it caused by xcc compiler, will check the exact error and try to fix it.